cmd/kubeadm/app/constants/constants_test.go

```diff
@@ -275,3 +275,71 @@ func TestGetSkewedKubernetesVersionImpl(t *testing.T) {
 		})
 	}
 }
+
+func TestGetAPIServerVirtualIP(t *testing.T) {
+	var tests = []struct {
+		name, svcSubnet, expectedIP string
```

Comment:
Now I'm curious, what happens if you pass a subnet like?

"10.96.0.12/24", it should give 10.96.0.1, or it fails the subnet parsing?